examples/stress_tests/many_text2d.rs

```
@@ -155,7 +155,9 @@ fn setup(mut commands: Commands, font: Res<FontHandle>, args: Res<Args>) {
 
 // System for rotating and translating the camera
 fn move_camera(time: Res<Time>, mut camera_query: Query<&mut Transform, With<Camera>>) {
-    let mut camera_transform = camera_query.single_mut();
+    let Ok(mut camera_transform) = camera_query.get_single_mut() else {
+        return;
+    };
```

Could the signature for `move_camera` not be changed to return `Result`, allowing the use of `?` here too?

---

Yes, but this is an example of a case where silently failing and just returning is always correct. You never want to panic here.